enterprise/backend/test/metabase_enterprise/advanced_permissions/api/group_manager_test.clj

```diff
@@ -1,4 +1,4 @@
-(ns metabase-enterprise.advanced-permissions.api.group-manager-test
+(ns ^{:mb/tag :mb.test/once} metabase-enterprise.advanced-permissions.api.group-manager-test
```

Comment:
Could this work if we just did `^:mb.test/once`? It's a lot easier to type

Reply:
Yeah i can do that. Originally it was like this but in making it more like tags I figured this kept a single place in the meta to look. I'll switch it back to this style.